web/src/routes/landing/components/CitySelector.js

```diff
@@ -49,8 +49,17 @@ class CitySelector extends React.PureComponent<PropsType> {
     }
   }
 
+  // Landkreis should come before Stadt
+  sort(cities: Array<CityModel>): Array<CityModel> {
+    return cities.sort(
+      (a, b) =>
+        a.sortingName.localeCompare(b.sortingName) || (a.prefix && b.prefix ? a.prefix.localeCompare(b.prefix) : 0)
```

Comment:
This does not handle comparison of prefix and no prefix, right? I.e. what would happen if there would be `Augsburg` and `Stadt Augsburg`. How about
```suggestion
        a.sortingName.localeCompare(b.sortingName) || (a.prefix || '').localeCompare(b.prefix || ''))
```
I think that should handle all cases correctly.

Reply:
Great idea. Didn't think about this